packages/fxa-auth-server/lib/server.js

```diff
@@ -427,6 +431,15 @@ async function create(log, error, config, routes, db, statsd) {
 
   server.auth.strategy('pubsub', 'jwt', pubsubAuth.strategy(config));
 
+  await server.register([
+    Inert,
```

Comment:
What's `Inert` and `Vision` doing? Seems like a new dependency we're only using once here? (might could use a comment above unless I'm just missing something obvious)